### DISCUSSION TITLE
Opnform setup with docker.

### DISCUSSION THREAD

**Discussion by linus5304** [OpnForm/OpnForm#208] [General]
Can I get the full functions of the pro version of opnform when I run it with docker ? 
- [ ] Must I have to trigger the stripe payment and will it work with a test account ?

**Comment by biohackinginternational** [comment]
Same, the pro features are not available on the self-hosted version!

**Comment by JhumanJ** [comment]
Hi there, fixed in https://github.com/JhumanJ/OpnForm/commit/00e98f6bc6c982d5f4adc49c40a44eef4d71ebd7. To fix locally, just make sure to remove the following from your `.env` or `.env.docker` file:
```
STRIPE_KEY=
STRIPE_SECRET=
```
